pom.xml

```diff
@@ -118,8 +118,6 @@
         </dependency>
     </dependencies>
 
-
-
     <build>
         <resources>
             <resource>
```

Comment:
Sorry for the change, test some dependencies for the import but delete all my test